api/envoy/config/trace/v3/zipkin.proto

```diff
@@ -65,4 +65,8 @@ message ZipkinConfig {
   // Determines the selected collector endpoint version. By default, the ``HTTP_JSON_V1`` will be
   // used.
   CollectorEndpointVersion collector_endpoint_version = 5;
+
+  // Optional hostname to use when sending spans to the collector_cluster. Useful for collectors
+  // that require a specific hostname. Defaults to `collector_cluster` above.
```

Comment:
We usually link this to the actual field.

Reply:
Done. I did something similar to the link I added in the release notes, let me know if that's what you had in mind.